BPOR/BPOR.Rms/Views/Campaign/Success.cshtml

```diff
@@ -0,0 +1,56 @@
+@using BPOR.Rms.Models.Email
+@using BPOR.Domain.Enums;
+@model BPOR.Rms.Models.Email.EmailSuccessViewModel
+
+@{
+    ViewData["Title"] = $"{Model.ContactMethod} campaign success - Volunteer Registry - Be Part of Research";
+}
+
+@if (!string.IsNullOrWhiteSpace(Model.StudyName))
+{
+    @if (Model.ContactMethod == ContactMethods.Email)
+    {
+        <div class="govuk-grid-row">
+            <div class="govuk-grid-column-one-half-from-desktop">
+                <div class="govuk-panel govuk-panel--confirmation">
+                    <h1 class="govuk-panel__title">
+                        Request to send emails for @Html.DisplayFor(model => model.StudyName) has been submitted to GOV.UK Notify
+                    </h1>
+                </div>
+            </div>
+        </div>
+    }
+    @if (Model.ContactMethod == ContactMethods.Letter)
+    {
+
+        <div class="govuk-grid-row">
+            <div class="govuk-grid-column-one-half-from-desktop">
+                <div class="govuk-panel govuk-panel--confirmation">
+                    <h1 class="govuk-panel__title">
+                        The letter campaign request for @Html.DisplayFor(model => model.StudyName) has been sent to GOV UK Notify
+                    </h1>
+                </div>
+            </div>
+        </div>
+    }
+}
+else
+{
+    <div class="govuk-grid-row">
+        <div class="govuk-grid-column-one-half-from-desktop">
+            <div class="govuk-panel govuk-panel--confirmation">
+                <h1 class="govuk-panel__title">
+                    The @Model.ContactMethod.ToString().ToLower() campaign has been sent
```

Comment:
incorrect success message.
see AC 5 
https://nihr.atlassian.net/browse/CRNCC-2394

`The letter campaign request has been sent to GOV UK Notify`

Reply:
@kevinofficer I have parameterised the view as per the technical breakdown to "maintain existing layout, appearance, and wording when email is selected". Is this the wrong approach?